README.md

```diff
@@ -161,3 +161,52 @@ There are three ways of creating a template.
     ![Image of default template](https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/_assets/SavedList.png)
 
 6. Modify report as you wish and click "Advanced Editor" button from the menu. Copy all contents and create a file like "your custom template name.template". Please make sure file name ends with '.workbook'.
+
+## How to associate any existing template to new category in Workbooks
+We are now supporting associating any exiting templates to new category (virtual category). Previously, a category is always associated with templates by a folder structure but this requires to create physical folder structure which requires copying existing templates. This would introduce a lot of maintain overhead of updating duplicated templates.
```

Comment:
maintainance

Reply:
(GitHub doesn't seem to do a good job of marking what was highlighted when you add a comment. :(  "maintain" in "this would introduce a lot of maintain" should be "maintenance")